lib/srv/db/common/reporting.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import "fmt"
+
+// ReporterMetadataFromProxyCtx returns string suitable for passing as a value for "service_metadata" label in
+// metrics "authenticated_active_connections" and "authenticated_accepted_connections_total"
+// declared by ingress.Reporter.
+func ReporterMetadataFromProxyCtx(proxyCtx *ProxyContext) string {
```

Comment:
The comment does not mention that if the list of servers is greater than 1, then only the first name is used.

Reply:
Fixed.